CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Client factories for Buzz.
 
+### Changed
+
+- Guzzle 6 client now uses the new `createWithConfig` named constructor
```

Comment:
this is very technical. i think for the user, its more relevant to know what that means:

- Guzzle 6 client is now created according to the Httplug specifications with automated minimal behaviour. Make sure you configure the Httplug plugins as needed, for example if you want to get exceptions for failure HTTP status codes.